javascript.js

```diff
@@ -0,0 +1,20 @@
+// Set variavles
+var user = []; //inline coment
+
+/**
```

Comment:
Typo inside comment in token 'Adds'. Possible corrections:dds (14),adds (14),odds (0)